charts/newrelic-infrastructure-beta/templates/ksm/deployment.yaml

```diff
@@ -0,0 +1,206 @@
+{{- if and (.Values.ksm.enabled) (.Values.enableLinux) }}
```

Comment:
I understand the motivation for`.Values.enableLinux` in other components, but not here: KSM scraper should not care about windows or linux.

I think we should completely get rid of any windows reference for this chart, and add it later if required. The old chart with windows-specific clauses will still be there.